kaspresso/src/main/kotlin/com/kaspersky/kaspresso/params/AutoScrollParams.kt

```diff
@@ -9,7 +9,7 @@ import junit.framework.AssertionFailedError
  * The class that holds all the necessary for [com.kaspersky.kaspresso.autoscroll.AutoScrollProviderImpl] and
  * [com.kaspersky.kaspresso.autoscroll.WebAutoScrollProviderImpl] parameters.
  */
-class AutoScrollParams private constructor(
+class AutoScrollParams(
```

Comment:
Hi @Starmel!
Thank you for your contribution!
Observing other such classes (`ContinuouslyParams.kt`, `FlakySafetyParams.kt`), I suggest adding a factory method instead of making the constructor public.